cobl2/models.py

```diff
@@ -76,6 +76,12 @@ class CognateClass(CustomModelMixin, Cognateset):
     comment = sa.Column(sa.Unicode)
     justification = sa.Column(sa.Unicode)
     revised_by = sa.Column(sa.Unicode)
+    proposedAsCognateTo_pk = sa.Column(sa.Integer, sa.ForeignKey('cognateclass.pk'))
+    proposedAsCognateTo_rel = sa.orm.relationship(
+        'CognateClass',
+        foreign_keys=[proposedAsCognateTo_pk],
+        backref=sa.orm.backref('proposedAsCognateTo', remote_side=[pk]))
+    proposedAsCognateToScale = sa.Column(sa.Integer)
```

Comment:
Considering that `proposedAsCognateToScale` seems to be metadata for the `proposedAsCognateTo_rel` relation, the more natural way to model this would be a many-to-many relation with an association table, and `scale` as column of this association table.

Reply:
there're only scale factors from 1 to 4 if proposedAsCognateTo_pk exists - ie one would create an association table with pks from 1 to 4, with scales 1 to 4 - but as for data clarity I'll change this later on

Reply:
Oh, no, the association table should be as follows:
```python
class ProposedCognates():
    cc1 = Column(Integer, ForeignKey('cognateclass.pk'))
    cc2 = Column(Integer, ForeignKey('cognateclass.pk'))
    scale = Column(Integer)
```

Reply:
With the current model, there can only be one proposal per cognate class, right? While this is presumably ok, because we won't get more proposals from an equally limited database, it looks a bit weird, in particular with additional metadata.